### DISCUSSION TITLE
focusTab suddenly does not work anymore

### DISCUSSION THREAD

**Discussion by roytouw** [johnlindquist/kit#1188] [Q&A] [answered]
I have made a very small script to open a specific tab in Chrome:

```
// Name: Open Chat GPT Chat
// Shortcut: cmd esc

focusTab("https://chat.openai.com/chat");
```
But all of a sudden this does no longer work, not via the shortcut and also not manually activating the script. Any ideas why this suddenly stopped working? I have the tab open in Chrome, just as before.

I'm running Chrome on a MacOS.

**Comment by johnlindquist** [comment, marked as answer]
@roytouw you can find the source for "focusTab" in ~/.kit/lib/browser.js

You'll see it needs to fire off Applescript to be able to run.

This usually requires "System Events" permissions (on my phone, I think that's right...). Chrome may have updated and you might need to reset the permissions and make sure Kit.app can properly fire off Applescript to control other apps

**Reply by roytouw** [reply]
Thank you, it appears to have been a permissions issue.
